src/loaders/facilityLoader.js

```diff
@@ -80,11 +82,8 @@ const facilityLoader = async (config) => {
 
         if (!associatedGeometries.length) {
```

Comment:
Similar to EarthEngineLoader.js L65, we get alerts for both OU missing geom and missing associated geom.
![image](https://github.com/user-attachments/assets/3803327d-66d9-4a31-bb48-39b2af594b95)

Reply:
This also was on purpose - to avoid logic changes.